mapie/regression/time_series_regression.py

```diff
@@ -454,16 +454,29 @@ def predict(
               - [:, 0, :]: Lower bound of the prediction interval.
               - [:, 1, :]: Upper bound of the prediction interval.
         """
+        alpha = self.transform_confidence_level_to_alpha_array(confidence_level)
         if alpha is None:
             super().predict(
                 X, ensemble=ensemble, alpha=alpha, optimize_beta=optimize_beta,
                 **predict_params
             )
-
         if self.method == "aci":
             alpha = self._get_alpha(alpha)
 
         return super().predict(
             X, ensemble=ensemble, alpha=alpha, optimize_beta=optimize_beta,
             allow_infinite_bounds=allow_infinite_bounds, **predict_params
         )
+
+    # The public API changed from alpha to confidence_level.
+    # TODO: refactor this class to use confidence_level everywhere
+    @staticmethod
+    def transform_confidence_level_to_alpha_array(
```

Comment:
Let's rename it to indicate it is private: `_transform_confidence_level_to_alpha_array`